providers/implementations/signature/dsa.c

```diff
@@ -331,13 +370,13 @@ static int dsa_get_ctx_params(void *vpdsactx, OSSL_PARAM *params)
         return 0;
 
     p = OSSL_PARAM_locate(params, OSSL_SIGNATURE_PARAM_DIGEST_SIZE);
```

Comment:
If we can now support a digest without a digest size - do we even need digest size at all? (I think this is new in master)

Reply:
I have wondered too, but figured that there are hash functions that can produce varying sizes according to user desire, so why not? They are not accepted today, but could in the future.

I would have no remorse removing it if you prefer.

Reply:
@mattcaswell ?

Reply:
I'd say lets remove it

Reply:
Okie

Reply:
Done